docs/topics/modules/using_guided_hierarchy_to_create_work_item.adoc

```diff
@@ -1,7 +1,7 @@
 [id="using_guided_hierarchy_to_create_work_item"]
 = Using the guided hierarchy to create a work item
 
-NOTE: The space template you select determines the guided work item type hierarchy.
+NOTE: The development process you select, while creating the Space, determines the guided work item type hierarchy.
```

Comment:
-> space

Reply:
This section has been changed and taken care of through #309